src/generateRenderer/index.js

```diff
@@ -0,0 +1,33 @@
+const Winnow = require('winnow')
+const { renderRenderers } = require('../templates')
+
+module.exports = generateRenderer
+
+/**
+ * processes params based on generate renderer params
+ *
+ * @param {object} data
+ * @param {object} params
+ * @param {function} callback
+ */
+function generateRenderer (data = {}, params = {}) {
+  try {
```

Comment:
Agreed. However, I currently have caught errors return `{}` so that something is returned. My tests check for `{}` too. Removing this causes a bunch of undefined errors in various places. How do you think I should handle these?